Lib/concurrent/futures/process.py

```diff
@@ -249,6 +252,8 @@ def _process_worker(call_queue, result_queue, initializer, initargs):
         # open files or shared memory that is not needed anymore
         del call_item
 
+        # increment idle process count after worker finishes job
+        idle_worker_semaphore.release()
```

Comment:
I considered having this in `_ExecutorManagerThread.run()` at first (both technically work), but I think having it in `_process_worker` instead will result in more immediate reuse of the workers, which helps to further minimize idle CPU time.

Reply:
I ended up needing to release the semaphore in `_ExecutorManagerThread.run()`, see comments below for more information.